frontend/institution/institutionService.js

```diff
@@ -18,9 +18,9 @@
             return deferred.promise;
         };
 
-        service.searchInstitutions = function searchInstitutions(value, state) {
+        service.searchInstitutions = function searchInstitutions(value, state, type) {
             var deferred = $q.defer();
-            $http.get("/api/search/institution?value=" + '"' + value + '"' + "&state=" + state).then(function success(response) {
+            $http.get("/api/search/institution?value=" + value + "&state=" + state + "&type=" + type).then(function success(response) {
```

Comment:
usa o padrão de query params do angular 

https://stackoverflow.com/questions/13760070/angularjs-passing-data-to-http-get-request